saml-core/src/main/java/org/keycloak/saml/processing/core/parsers/saml/assertion/SAMLAttributeValueParser.java

```diff
@@ -105,7 +105,23 @@ public Object parse(XMLEventReader xmlEventReader) throws ParsingException {
             return StaxParserUtil.getElementText(xmlEventReader);
         }
 
-        throw logger.parserUnknownXSI(typeValue);
+        // KEYCLOAK-18417: Simply ignore unknown types
+        logger.warn("Skipping attribute value of unsupported type " + typeValue);
```

Comment:
Please use 
```suggestion
        logger.debugf("Skipping attribute value of unsupported type %s", typeValue);
```

Reply:
In this class, a PicketLinkLogger is used, it has no method debugf(). I do not want to change too much of the existing code, so I'd rather go with ` logger.debug("Skipping attribute value of unsupported type " + typeValue);`

Reply:
Apologies, this was omission on my end. You're right.